.env

```diff
@@ -90,3 +90,4 @@ _APP_LOGGING_CONFIG=
 _APP_REGION=default
 _APP_DOCKER_HUB_USERNAME=
 _APP_DOCKER_HUB_PASSWORD=
+_APP_MAINTENANCE_RETENTION_SCHEDULES=86400
```

Comment:
Just a formatting thing. Can you add this variable near all the other maintenance variables ?